app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+#
+class User < ActiveRecord::Base
+  has_secure_password
+  has_one :profile
```

Comment:
No reason to have a separate profile relation. Profile is currently an empty model... we should remove it and this relation and add fields to user as needed until there is a compelling reason to add complexity.